src/components/MainNav/MyProfile/MyProfile.js

```diff
@@ -47,6 +55,38 @@ export default class MyProfile extends Component {
     this.getProfileImage = this.getProfileImage.bind(this);
   }
 
+  componentDidMount() {
+    const { modules } = this.props;
+    const navLinks = modules.settings.map(({ links }) => links);
+
+    if (navLinks && Array.isArray(navLinks)) {
+      const links = navLinks[0].userDropdown;
+      links.forEach((link, index) => {
+        if (this.props.stripes.user) {
+          if (!link.check) {
+            this.createLink(link, index);
+          } else if (typeof userDropodownChecks[link.check] === 'function') {
+            userDropodownChecks[link.check]().then((canShow) => {
+              if (canShow) {
+                this.createLink(link, index);
+              }
+            });
+          }
+        }
+      });
+    }
+  }
+
+  createLink(link, index) {
+    const buttonId = 'clickable-menuItem' + index;
```

Comment:
Let`s use es6 template strings. Change to

```
const buttonId = `clickable-menuItem${index}`;
```

Reply:
This is the kind of thing that ESLint usually picks up. @natali-zaitseva, did you run `yarn lint` on your modified code?